libs/utils/decorate.js

```diff
@@ -65,6 +65,14 @@ export function decorateIconArea(el) {
   });
 }
 
+function elContainsText(el) {
+  return !![...el.childNodes].filter((childNode) => {
+    const { innerText, nodeType, textContent } = childNode;
+    if (nodeType === Node.ELEMENT_NODE) return innerText.trim() !== '';
+    return nodeType === Node.TEXT_NODE && textContent.trim() !== '';
+  }).length;
+}
```

Comment:
This can be simplified, something along the lines of:
```suggestion
const elContainsText = el => 
  [...el.childNodes].some((childNode) => 
    (childNode.nodeType === Node.ELEMENT_NODE && childNode.innerText.trim() !== '') ||
    (childNode.nodeType === Node.TEXT_NODE && childNode.textContent.trim() !== '')
  );
```

Reply:
Thanks, yeah I like that. I have updated it to use the `some`